assets/src/edit-story/elements/text/index.js

```diff
@@ -47,6 +47,8 @@ export const defaultAttributes = {
     vertical: 0,
     horizontal: 0,
   },
+  lockPadding: true,
+  lockAspectRatio: true,
```

Comment:
This could probably be added to the `SHARED_DEFAULT_ATTRIBUTES` since all the elements can have that one, not just Text.

Reply:
Yes, I'm on it. Thank you

Reply:
@miina We can have lockAspectRatio on `SHARED_DEFAULT_ATTRIBUTES`, but lockPadding is supposed to be Text element only I think.

Reply:
Then we don't need it here also, do we?

Reply:
`lockAspectRatio` doesn't make a lot of sense for text. Only width can be specified for text - height is auto-calculated.

Reply:
Well, we currently allow users to enter width and height directly and font size will be adjusted accordingly. As long as we do that, locking aspect ratio makes sense, no?

Reply:
Yeah. Let's put it for now. It will be too much to address all text nuances for this.